client/java/generator/.classpath

```diff
@@ -26,7 +26,6 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-13/"/>
-	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-13/"/>	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
```

Comment:
why do we need this file?

Reply:
Not sure, probably we should remove it. 
Anyway, change to this file was commited accidentally, removed it.